src/Symfony/Bundle/FrameworkBundle/Tests/Templating/TemplateNameParserTest.php

```diff
@@ -86,17 +85,15 @@ public function testParseValidNameWithNotFoundBundle()
     /**
      * @group legacy
      * @dataProvider provideAbsolutePaths
-     * @requires function Symfony\Bridge\PhpUnit\ErrorAssert::assertDeprecationsAreTriggered
+     * @expectedDeprecation Absolute template path support is deprecated since Symfony 3.1 and will be removed in 4.0.
```

Comment:
We will need to update tests in older branches too (see https://github.com/symfony/symfony/blob/3.1/src/Symfony/Bundle/FrameworkBundle/Tests/Templating/TemplateNameParserTest.php#L89, for an example). Updating them should be done in separate PRs (except for deprecations introduced in 3.2).